Plugin crashes when opening project

Importing this [project](https://github.com/dinosaurwithakatana/freight) into Android Studio 2.1.1 with sqldelight installed throws the following error and aborts startup.


```
com.intellij.diagnostic.PluginException: null cannot be cast to non-null type kotlin.String [Plugin: com.squareup.sqldelight]
com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: null cannot be cast to non-null type kotlin.String [Plugin: com.squareup.sqldelight]
	at com.intellij.ide.plugins.PluginManager.handleComponentError(PluginManager.java:242)
	at com.intellij.openapi.components.impl.PlatformComponentManagerImpl.handleInitComponentError(PlatformComponentManagerImpl.java:43)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:536)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:123)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:107)
	at com.intellij.openapi.module.impl.ModuleImpl.init(ModuleImpl.java:88)
	at com.intellij.openapi.module.impl.ModuleManagerImpl$ModuleModelImpl.initModule(ModuleManagerImpl.java:790)
	at com.intellij.openapi.module.impl.ModuleManagerImpl$ModuleModelImpl.newModule(ModuleManagerImpl.java:716)
	at com.intellij.openapi.module.impl.ModuleManagerImpl$ModuleModelImpl.newModule(ModuleManagerImpl.java:703)
	at com.intellij.openapi.externalSystem.service.project.AbstractIdeModifiableModelsProvider.newModule(AbstractIdeModifiableModelsProvider.java:109)
	at com.intellij.openapi.externalSystem.service.project.manage.ModuleDataService.createModules(ModuleDataService.java:99)
	at com.intellij.openapi.externalSystem.service.project.manage.ModuleDataService.importData(ModuleDataService.java:85)
	at com.intellij.openapi.externalSystem.service.project.manage.ProjectDataManager.doImportData(ProjectDataManager.java:254)
	at com.intellij.openapi.externalSystem.service.project.manage.ProjectDataManager.importData(ProjectDataManager.java:151)
	at com.intellij.openapi.externalSystem.service.project.manage.ProjectDataManager.importData(ProjectDataManager.java:197)
	at com.android.tools.idea.gradle.util.Projects$1$1$1.run(Projects.java:136)
	at com.intellij.openapi.roots.impl.ProjectRootManagerImpl.mergeRootsChangesDuring(ProjectRootManagerImpl.java:321)
	at com.android.tools.idea.gradle.util.Projects$1$1.run(Projects.java:132)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:1010)
	at com.android.tools.idea.gradle.util.Projects$1.run(Projects.java:128)
	at com.intellij.util.ui.UIUtil.invokeAndWaitIfNeeded(UIUtil.java:2382)
	at com.android.tools.idea.gradle.util.Projects.populate(Projects.java:121)
	at com.android.tools.idea.gradle.project.ProjectSetUpTask$2.run(ProjectSetUpTask.java:106)
	at com.intellij.ide.startup.impl.StartupManagerImpl$7.run(StartupManagerImpl.java:372)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:337)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:321)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:866)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:654)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:381)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: com.intellij.diagnostic.PluginException: null cannot be cast to non-null type kotlin.String [Plugin: com.squareup.sqldelight]
	... 43 more
Caused by: kotlin.TypeCastException: null cannot be cast to non-null type kotlin.String
	at com.squareup.sqldelight.intellij.SqlDelightModuleComponent$initComponent$1.invoke(SqlDelightModuleComponent.kt:44)
	at com.squareup.sqldelight.intellij.SqlDelightModuleComponent$initComponent$1.invoke(SqlDelightModuleComponent.kt:34)
	at com.squareup.sqldelight.intellij.util.GradleUtilKt$iterateClasspath$1.onPsiFileAvailable(GradleUtil.kt:19)
	at com.android.tools.idea.gradle.parser.GradleGroovyFile$1.run(GradleGroovyFile.java:167)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:946)
	at com.android.tools.idea.gradle.parser.GradleGroovyFile.reload(GradleGroovyFile.java:172)
	at com.android.tools.idea.gradle.parser.GradleBuildFile.reload(GradleBuildFile.java:48)
	at com.android.tools.idea.gradle.parser.GradleGroovyFile.<init>(GradleGroovyFile.java:73)
	at com.android.tools.idea.gradle.parser.GradleBuildFile.<init>(GradleBuildFile.java:62)
	at com.squareup.sqldelight.intellij.util.GradleUtilKt$iterateClasspath$1.<init>(GradleUtil.kt:13)
	at com.squareup.sqldelight.intellij.util.GradleUtilKt.iterateClasspath(GradleUtil.kt:13)
	at com.squareup.sqldelight.intellij.SqlDelightModuleComponent.initComponent(SqlDelightModuleComponent.kt:43)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:518)
	... 40 more
```

--- a/sqldelight-studio-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/SqlDelightModuleComponent.kt
+++ b/sqldelight-studio-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/SqlDelightModuleComponent.kt
@@ -41,8 +41,10 @@ class SqlDelightModuleComponent(val module: Module) : ModuleComponent {
   override fun getComponentName() = "SqlDelightModuleComponent"
   override fun initComponent() {
     iterateClasspath(module) { classpath ->
-      val classpathValue = BuildFileKeyType.STRING.getValue(classpath) as String
-      if (classpathValue == GradleBuildFile.UNRECOGNIZED_VALUE) return@iterateClasspath
+      val classpathValue = BuildFileKeyType.STRING.getValue(classpath) as? String
+      if (classpathValue == null || classpathValue == GradleBuildFile.UNRECOGNIZED_VALUE) {
+        return@iterateClasspath
+      }
 
       val gradleVersion = classpathValue.substringAfterLast(':')
 

